atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/knowledge/AbortedTransactionSoftCache.java

```diff
@@ -47,37 +46,34 @@ public AbortedTransactionSoftCache(
         this.futileTimestampStore = futileTimestampStore;
         this.knownConcludedTransactions = knownConcludedTransactions;
         this.patchyCacheRef = new AtomicReference<>();
-        this.autobatcher = Autobatchers.independent(consumer())
+        this.autobatcher = Autobatchers.independent(this::consumer)
```

Comment:
I guess there is some reconciling to be made with the base branch, but the changes here make sense. Though we still want to make this coalescing